### ISSUE TITLE
Motivation

### ISSUE THREAD

**Issue by webmaxru** [webmaxru/progressive-web-conf#3]

**Comment by webmaxru** [comment]
2018 is the #YearOfPWA - the year when all major browser vendors released at least basic PWA functionality. The web now has a unique chance to become a truly universal platform for all kinds of the apps on all kinds of devices - the term "web" is not about "surfing the websites while online" anymore.

We (enthusiasts, devrels, browser vendor reps)  advocate PWAs on many conferences - I believe, there was no a single web-related event without at least one session about this topic this year. We say "this is an exciting new world of building the cross-platform apps". At the same time, the question "should I choose a PWA way for building a new app _right here and right now_" is still far from having a clear, reasoned (with enough pros and cons) answer. [Web-, mobile-] developers are worrying about:

- What is the current level of support of PWA features in the browsers/platforms I target my app for? This is partially answered by resources like IsServiceWorkerReady, PWA Feature Detector.
- If and when the _feature I need to have_ to be delivered to the _platform I  need to support_. It's almost impossible to find the clear roadmap.
- How could I workaround the _current_ limitations of PWA support with the intention to remove these workarounds in the future? There are some interesting options many developers are not aware of.
- ...many more technical questions from the developers, and strategic questions from the business side.

The main point of organizing Progressive Web Conf (working title) is to confirm:
- PWA is the really efficient way to build the apps. We need to showcase the real life examples.
- It's safe to invest developer's time in learning and company's money in building a PWA. We are in the situation whrn only browser/platform vendors are eligible to say something about that.

Both points (especially, the second one) are not possible to reason about without _all_ browser vendors involved in the discussion.

**Comment by comp615** [comment]
The two main points of organizing sound very developer-centric to me. My summary would be it's a conference to learn about why PWAs work and how to build them (or transform your existing app). That's not a bad thing at all, and is actually quite valuable. I just wanted to point out there's many aspects I think could be valuable, maybe they can happen at the same event on different times, maybe not. But for discussion...

I'd be interested to hear what the browser vendors think, but from my perspective there's value in supporting alignment from the various vendors around ideas. For instance, as I talked with people at the recent conferences, it seems like there's a lot of questions about how to prompt installing a PWA. Chrome led with one approach, but should that be a standards approach? What did they learn or what would they change that might help when ATHS for desktop comes? Should it be a user controlled command instead of event? How should the UI/messaging align?

Moreover, maybe this happens, but I'd love to just align the vendors on a roadmap so things don't get crazy splintered and the PWA brand as a whole can progress independent (to an extent).

To mix the two, I also think there's value in discussing as a SMALL roundtable directly what devs view as problems or opportunities in PWAs with the vendors. Are we finding that we need an interface to detect the current OS theme color? More generally, do we need a better way to expose OS apis? Should network signal be the focus? How important is sharing images to apps? A smaller meeting could be more productive where vendors can ask these questions directly to apps and have a tight feedback cycle.

TL;DR: I think you illustrated what aspiring PWA developers would want to get out, but I'm interested also in hearing how the vendors themselves think they could benefit, and how someone who's bought into the PWA ecosystem already might benefit. Figuring out those three use cases, and which fit into this conference will make it even more successful.

**Comment by webmaxru** [comment]
@comp615 I share your concerns and ideas. Actually, I'm still unsure about the term "conference" in the naming. My initial idea was about facilitating offline discussions like this https://github.com/w3c/manifest/issues/627 by gathering the feedback from all the browser vendors and dev community.

Also, as Aaron Gustafson mentioned here https://twitter.com/AaronGustafson/status/994446815310200832 - it would be cool to gather browser folks for the _public_ discussion. Even if they don't share any extraordinary info and provide only some blurry roadmaps (developers don't like to prognose), the fact that they are _all together_ gathered at some event to present/argue/reason about PWA is (and I truly believe in that) a huge factor of trust for the whole dev community. There _never_ was a meeting (at least public one) like this before.

Anyway, it doesn't mean that the event will look like the meeting / panel / round table with browser vendors discussing and presenting and silent web developers around. We have to find a good balance on how to build the dialogue.

**Comment by webmaxru** [comment]
@comp615: more thoughts: https://github.com/webmaxru/progressive-web-conf/issues/2#issuecomment-390192418

**Comment by torgo** [comment]
@webmaxru: @thisisjofrank suggested an organizing committee. Can we establish such a committee? I would like to suggest yourself, two browser representatives and two developer community representatives.

**Comment by webmaxru** [comment]
@torgo: yes, I like the suggested structure and I'm ready to be a part of the committee! Could you or someone from your team take one of the browser reps place?
@thisisjofrank: do you have the wish and bandwidth to join the committee as a representative from dev community? (I don't know your background, but I suggest you are a developer interested in PWA, please correct me if I'm wrong)

**Comment by thisisjofrank** [comment]
I'd love to join. I'm on the Samsung Internet team and an organiser of codebar, other tech meet-ups and conferences.

**Comment by webmaxru** [comment]
@thisisjofrank I'm happy to have you on the team! Could you please also add yourself to the list of participants (this is not only the _team_ but a list of all who has shown in any form) by sending a PR to https://github.com/webmaxru/progressive-web-conf/blob/master/PARTICIPANTS.md
